cs3api4lab/tests/test_cs3_ocm_share_api.py

```diff
@@ -1,11 +1,12 @@
-from unittest import TestCase
+from unittest import TestCase, skip
 from cs3api4lab.api.cs3_file_api import Cs3FileApi
 from cs3api4lab.config.config_manager import Cs3ConfigManager
 from traitlets.config import LoggingConfigurable
 from cs3api4lab.api.cs3_ocm_share_api import Cs3OcmShareApi
 
 
-class TestCs3ShareApi(TestCase):
+@skip
```

Comment:
I supposed we're skipping this just for the moment and we will bring this back soon, no?

Reply:
yes